running-requests/prompt-blueprints.mdx

```diff
@@ -132,3 +132,48 @@ response = pl_client.run(
 
 print(response)
 ```
+
+
+## Using Image Variable
+
+The `image_variable` attribute allows you to dynamically insert a list of images into your prompt templates. This attribute is part of the `ImageContent` structure and can be used to include images based on a specified variable name.
+
+### Structure
+
+The `image_variable` is nested within the message content.
+
+```json
+{
+    “role”: “user”,
+    “content”: [
+        {
```

Comment:
This is confusing.. are we adding the image or the the image variable. What is image url?

Reply:
`image_variable` exists inside `ImageContent`. `image_url` can be used to reference images from a URL